### PR TITLE
metrics: Restore disk statistics

### PR DESCRIPTION
**Pull request by shohamc1**
Grafana preview

Screenshot 2024-03-18 at 6 39 55 PM

### CODE REVIEW

File: erigon-lib/common/disk/common.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import (
+	"context"
+	"time"
+
+	"github.com/ledgerwatch/log/v3"
+)
+
+func UpdateDiskStats(ctx context.Context, logger log.Logger) {
+	logEvery := time.NewTicker(180 * time.Second)
```
Comment (shohamc1) [inline erigon-lib/common/disk/common.go:11]: @AskAlexSharov I was wondering if 180 seconds is too long for this metric, should we shorten it? If yes, what would be suitable?

Comment (AskAlexSharov) [reply erigon-lib/common/disk/common.go:11]: fine for me

File: eth/backend.go [hunk 2]
```diff
@@ -490,6 +491,7 @@ func New(ctx context.Context, stack *node.Node, config *ethconfig.Config, logger
 
 	// setup periodic logging and prometheus updates
 	go mem.LogMemStats(ctx, logger)
+	go disk.UpdateDiskStats(ctx, logger)
```
Comment (AskAlexSharov) [inline eth/backend.go:494]: we have many binaries: `cmd/integration`, etc... need add it to them allso

**Review by AskAlexSharov — APPROVED** [review]

**Comment by taratorio** [comment]
relates to https://github.com/ledgerwatch/erigon/issues/8830

**Pull request merged**